codalab/apps/web/templates/web/worksheets/detail.html

```diff
@@ -70,6 +70,12 @@
                       <td><kbd>enter</kbd></td><td>Open current bundle or worksheet</td>
                     </tr>
                     <tr>
+                        <td><kbd>u</kbd></td><td>Paste UUID of current bundle or worksheet into console</td>
+                    </tr>
+                    <tr>
+                        <td><kbd>a</kbd></td><td>Paste run arguments of current bundle into console</td>
```

Comment:
not just run arguments - because it works for make bundles too.  Replace with 'paste arguments to recreate current bundle'.

Also, we should be consistent, and call 'console' the 'web terminal'.  Ideally, we would rename 'action bar' (it was a bar at some point), but not for this PR....